part-spec/power.json

```diff
@@ -0,0 +1,210 @@
+                "type": "string"
+            },
+            "orderableMPN": {
+                "description": "orderable manufacturer part number",
+                "type": "string"
+            },
+            "sourceDatasheetID": {
+                "description": "methods for identifying the human-readable source information for a digital datasheet",
+                "$ref": "#/$defs/sourceDatasheetID"
+            },
+            "digitalDatasheetID": {
+                "description": "methods for identifying the version of the digital datasheet",
+                "$ref": "#/$defs/digitalDatasheetID"
+            },
+            "status": {
+                "description": "production status of a component",
+                "examples": [
+                    "preProduction",
+                    "active",
+                    "EOL"
+                ],
+                "type": "string"
+            },
+            "regulatorTopology": {
+                "description": "switching voltage regulator topology",
+                "examples": [
+                    "buck",
+                    "boost",
+                    "buck-boost",
+                    "other"
+                ],
+                "type": "string"
+            },
+            "vinMin": {
+                "description": "minimum input voltage under which the part can be expected to operate properly",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "vinMax": {
+                "description": "maximum input voltage under which the part can be expected to operate properly",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "voutMin": {
+                "description": "minimum output voltage the part can regulate",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "voutMax": {
+                "description": "maximum output voltage the part can regulate",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "feedbackVoltage": {
+                "description": "voltage comparison point at the feedback node",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "maxLoadCurrent": {
+                "description": "maximum load current supported by regulator",
+                "comment": "units of amps",
+                "$ref": "#/definitions/unit"
+            },
+            "minLoadCurrent": {
+                "description": "minimum load current supported by regulator",
+                "comment": "units of amps",
+                "$ref": "#/definitions/unit"
+            },
+            "voutAccuracy": {
+                "description": "output voltage variation at no load",
+                "comment": "units of %",
```

Comment:
I think in some cases, the accuracy is also mV. We can remove the comment and only have the reference to unit definition to allow users to use % or Volts